server/config.go

```diff
@@ -115,42 +111,24 @@ func (c *Configuration) AddDomain(domain string) (err error) {
 	return
 }
 
-// EnrichCapabilities will update the capabilities with the appropriate service url
-func (c *Configuration) EnrichCapabilities(domain string) *paymail.CapabilitiesPayload {
-	capabilities := &paymail.CapabilitiesPayload{
-		BsvAlias:     c.Capabilities.BsvAlias,
-		Capabilities: make(map[string]interface{}),
-	}
-	for key, val := range c.Capabilities.Capabilities {
-		if w, ok := val.(string); ok {
-			capabilities.Capabilities[key] = GenerateServiceURL(c.Prefix, domain, c.APIVersion, c.ServiceName) + w
-		} else {
-			capabilities.Capabilities[key] = val
-		}
-	}
-	return capabilities
-}
-
 // GenerateServiceURL will create the service URL
-func GenerateServiceURL(prefix, domain, apiVersion, serviceName string) string {
-
-	// Require prefix or domain
+func GenerateServiceURL(prefix, domain, apiVersion, serviceName string) (string, error) {
```

Comment:
Shouldn't be moved to capabilities.go?

Bonus question- are you sure there are benefits of using builder for such small string? (I'm just curious)

Reply:
> Shouldn't be moved to capabilities.go?

It's used only in `capabilities.go`, but since it's a public function whose functionality extends beyond the scope of capabilities, I decided not to move it from its original location. However, if you would like me to move it, please let me know, as it doesn't make much difference to me

> Bonus question- are you sure there are benefits of using builder for such small string? (I'm just curious)

I made a benchmark. The results shows that the `old` implementation is slightly better when *short* strings are concerned. For example for a string: `https://bux.4chain.space:1234/v1/bsvalias` (made of 41chars) the *old* implementation wins. 

![image](https://github.com/bitcoin-sv/go-paymail/assets/152964795/83cec300-ffd3-4545-8cbd-109f44d685ce)
![image](https://github.com/bitcoin-sv/go-paymail/assets/152964795/33f1fed8-de0f-4ea9-8371-308b9b86e100)


Another question is whether this function really matters in terms of performance. I've observed that this function must be called with every `/.well-known/bsvalias` request, and it cannot be calculated during the initialization stage due to the arg:host, which is not constant. That's why I decided to refactor it in the first place.